src/components/RevealCarousel.js

```diff
@@ -1,5 +1,5 @@
 import React, { Component } from 'react'
```

Comment:
I think we should probably re-image/re-dev this entire component. 
It's sprawling and a big way.